ramls/item.json

```diff
@@ -296,6 +296,45 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
```

Comment:
There needs to be a description for the object property as well as the member properties, please can one be added.

Reply:
Done.